twisted/conch/ssh/keys.py

```diff
@@ -782,18 +783,23 @@ def public(self):
         return Key(self._keyObject.public_key())
 
 
-    def fingerprint(self):
+    def fingerprint(self, hash_function=sha256):
```

Comment:
I disagree with @adiroiban here.  The standard response is to not change
behavior of existing code, but I think someone who is more familiar with security
should look at this.
Changing the default behavior of the fingerprint function to sha256 is a reasonable
change as long as it is documented and in the release notes.